components/raftstore-v2/src/operation/ready/mod.rs

```diff
@@ -680,7 +680,7 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             return;
```

Comment:
update the argument name (from, to) to (old_index, new_index).

Reply:
Done